test/functional/p2p_invalid_block.py

```diff
@@ -133,5 +136,19 @@ def run_test(self):
         self.log.info("Test inflation by duplicating input")
         peer.send_blocks_and_test([block4], node, success=False,  reject_reason='bad-txns-inputs-duplicate')
 
+
+        self.log.info("Test accepting identical block after rejecting it due to a future timestamp.")
+
+        t = int(time.time())
+        node.setmocktime(t)
+        block = create_block(tip, create_coinbase(height), t + 7201)
+        block.hashMerkleRoot = block.calc_merkle_root()
+        block.solve()
+        # NOTE: Need force_send because the block will get rejected without a getdata otherwise
```

Comment:
This is a comment, there's no need to say "NOTE"
```suggestion
        # Need force_send because the block will get rejected without a getdata otherwise
```

Reply:
Removed!

Reply:
Hm, I don't think it's been removed?